packages/cyberstorm/src/components/Button.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { MouseEventHandler, ReactNode } from "react";
+import styles from "./componentStyles/Button.module.css";
+
+export interface ButtonProps {
+  label?: string;
+  leftIcon?: ReactNode;
+  rightIcon?: ReactNode;
+  buttonStyle?: string;
+  onClick?: MouseEventHandler<HTMLButtonElement>;
+}
+
+/**
+ * Cyberstorm Button component
+ */
+export const Button: React.FC<ButtonProps> = (props) => {
+  const { label, leftIcon, rightIcon, buttonStyle, onClick } = props;
+  const stylesImport = styles;
+  const additionalStyle = buttonStyle
+    ? " button__" + buttonStyle
+    : " button__default";
+
+  return (
+    <button
+      type="button"
+      className={"button" + additionalStyle}
```

Comment:
Redeclaring this here so it doesn't get lost in Discord: this approach doesn't work in a standalone React package like it does in Sertra which uses Next.js. We need some tooling to support importing CSS modules into TS. Currently this will produce a static string class name, which means the styles may leak to or be overwritten from elsewhere. We should use JS to access the correct styles from the imported `styles`, which should add a unique hash to prevent the issues.

Assuming a ticket for this exists and the work continues soonish.